endoreg_db/services/video_import.py

```
@@ -372,294 +865,147 @@ def _ensure_frame_cleaning_available(self):
-                self.logger.info(f"Retrieved processor ROI information: endoscope_roi={endoscope_roi}")
-            else:
-                self.logger.warning(f"No processor found for video {self.current_video.uuid}, proceeding without ROI masking")
-                
-        except Exception as e:
-            self.logger.error(f"Failed to retrieve processor ROI information: {e}")
-            # Continue without ROI - don't fail the entire import process
-        
-        return processor_roi, endoscope_roi
-
-    def _perform_frame_cleaning(self, FrameCleaner, processor_roi, endoscope_roi):
+    def _perform_frame_cleaning(self, FrameCleaner, endoscope_data_roi_nested, endoscope_image_roi):
```

Pylint (4.0.1)

[refactor] Too many local variables (24/15)

(R0914)